arch/arm64/src/common/arm64_boot.c

```diff
@@ -78,7 +78,7 @@ void arm64_boot_el3_init(void)
   reg = 0U;               /* Reset */
   reg |= SCR_NS_BIT;      /* EL2 / EL3 non-secure */
   reg |= (SCR_RES1 |      /* RES1 */
-  #ifdef CONFIG_ARCH_BOOT_EL3
+#if (CONFIG_ARCH_ARM64_EXCEPTION_LEVEL == 3)
```

Comment:
remove ALL ()

Reply:
You mean from all:
#if (CONFIG_ARCH_ARM64_EXCEPTION_LEVEL == 3)
into form:
#if CONFIG_ARCH_ARM64_EXCEPTION_LEVEL == 3
?

Reply:
yes.

Reply:
Right, fixed now!